components/modals/viewOrder.js

```diff
@@ -0,0 +1,156 @@
+import React, {useState, useEffect, useRef} from "react";
+import { makeStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import Modal from '@material-ui/core/Modal';
+import {connect, useDispatch} from 'react-redux';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import { updateStatus } from '../../redux/actions';
+
+
+function getModalStyle() {
+    const top = 50;
+    const left = 50;
+    return {
+        top: `${top}%`,
+        left: `${left}%`,
+        transform: `translate(-${top}%, -${left}%)`,
+    };
+}
+
+const useStyles = makeStyles(theme => ({
+    modal: {
+        // display: 'grid',
```

Comment:
can you please remove these commented code ? if not needed. and i see this being used again and again can you make it in way that it can be reused again